src/server/dataSource.js

```diff
@@ -13,6 +13,11 @@ var fluid = require("infusion");
 require("kettle");
 
 var sjrk = fluid.registerNamespace("sjrk");
+fluid.registerNamespace("sjrk.storyTelling.server.dataSource.couch.story");
+
+// This is the data schema version for the project.
+// All new stories will have their version set to this value.
+sjrk.storyTelling.server.dataSource.couch.story.schemaVersion = "0.4.0";
```

Comment:
I feel it probably makes more sense to separate the version of the data model from the version of the SJRK tool since not every SJRK release will include a data model update.

It's also odd to see the schema version bumps from undefined to 0.4.0.

@BlueSlug, when you have time, It would be nice to have a data model documentation with data structure for every schema version including undefined.

Reply:
This is a fair point. I had also considered that we should keep the schema version and the tool version separate, and I think I mentioned it during our chat yesterday, though I'm not sure what we should be starting at.

I suppose, since it's an arbitrary number and it doesn't (and in some ways shouldn't) conform to semantic versioning rules, we could start with "1.0", or we could equally use "0.1" since this is a work in progress. Do you have any preference or suggestion? I would want to rename the migration script accordingly, to something like `migrate_unversioned_to_0.1.js`

For documentation of the data model versions, `undefined` consists of all previous states/versions, so it's a little challenging and potentially confusing to list everything. Would it be reasonable to document it as the fields present in v0.3.0 of the tool?

Reply:
I'm fine with either "1.0.0" or "0.0.1", whichever makes more sense to you and @jobara. I would suggest we follow the convention and standard to use 3 numbers in version representing "MAJOR.MINOR.PATCH".

Renaming the migration folder and script to "migrate_unversioned_to_0.0.1" looks great too.

Your point of `undefined` consisting of all previous states/versions makes total sense. I agree it's reasonable to document as of what's in v0.3.0 of the tool.